core/src/main/scala/kafka/server/AddPartitionsToTxnManager.scala

```diff
@@ -86,7 +86,7 @@ class AddPartitionsToTxnManager(
   private val verificationFailureRate = metricsGroup.newMeter(VerificationFailureRateMetricName, "failures", TimeUnit.SECONDS)
   private val verificationTimeMs = metricsGroup.newHistogram(VerificationTimeMsMetricName)
 
-  def verifyTransaction(
+  def addOrVerifyTransaction(
```

Comment:
For this file, we should double check that there are no new errors to produce/offsetCommit that can be returned by the adding partition path. If there are, we may wan to consider converting them.

Reply:
Added a section in the replica manager callback to convert the error codes.